config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -94,7 +94,9 @@ default boolean isConsensusMigration() {
 
   OptionalLong getCancunTime();
 
-  OptionalLong getShandongBlockNumber();
+  OptionalLong getFutureTime();
```

Comment:
Maybe we should be more explicit and use 'futureEipTime' to distinguish from the fact that 'experimentTime' can also be used as a non-EIP toggle?

Reply:
yeah, FutureTime is a little too abstract. Maybe FutureForkTime? TBDForkTime?

Reply:
This is why I proposed `Pizza` and `Bagal` ("The Works Pizza" for Future and "Everything Bagel" for Experimental)